emails/migrations/0040_add_block_list_emails_fields.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.26 on 2022-02-09 21:34
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('emails', '0039_profile_auto_block_spam'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='domainaddress',
+            name='block_list_emails',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
+            model_name='relayaddress',
+            name='block_list_emails',
+            field=models.BooleanField(default=False),
+        ),
```

Comment:
We deploy a single node into the production cluster and then it runs migrations. So there's a short interval of time (seconds?) where the load balancer might route some requests to the single node with new code before it's done running the migration. We usually see a few seconds where a dozen or so requests throw DB-related errors. :/

Reply:
OK, that seems like we're a few infrastructure tweaks away from "migrations before code". I can hold my nose and take a few DB errors for now, unless you want to start splitting migration PRs across deploys.

Reply:
Nah, separating deploys from migrations to code deploys feels like it could be even more cumbersome or error-prone. Some day we can work with SRE to change up the deploy so it runs prod migrations before putting the new node into the load balancer rotation.